src/reducers/7.12/reducer.test.ts

```diff
@@ -50,8 +50,8 @@ describe('test the creation broker reducer', () => {
     const newState = artemisCrReducer(initialState, {
       operation: ArtemisReducerOperations.decrementReplicas,
     });
-    // default size is 1 decrementing should result of a size of 1
-    expect(newState.cr.spec.deploymentPlan.size).toBe(1);
+    // default size is 1 decrementing should result of a size of 0
+    expect(newState.cr.spec.deploymentPlan.size).toBe(0);
```

Comment:
Can you add a state manipulation in this test to try to decrement under 0 and make sure that the value doesn't go below 0 please?

```
    const newState = artemisCrReducer(initialState, {
      operation: ArtemisReducerOperations.decrementReplicas,
    });
    // decrementing from 0 should leave a value of zero
    expect(newState.cr.spec.deploymentPlan.size).toBe(0);
```